attestor/command/cli.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package pkg defines fns for running Scorecard checks on a Repo.
+
+package command
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/golang/glog"
+)
+
+type SignerMode string
+
+const (
+	CheckAndSign  SignerMode = "check-and-sign"
+	CheckOnly     SignerMode = "check-only"
+	BypassAndSign SignerMode = "bypass-and-sign"
+)
+
+var (
```

Comment:
Would highly recommend re-writing this using cobra.dev. The logic to parse and validate CLI flags is not something that's worth implementing and maintaining ourselves.